test/presubmit-tests.sh

```diff
@@ -183,8 +198,13 @@ unit_tests() {
 }
 
 integration_tests() {
-  warn "No integration tests to run"
-  return 0
+  (
+    set -eu -o pipefail
+
+    cd "$API_DIR"
+    api-e2e || return 1
```

Comment:
should we add a makefile target for it?

Reply:
Yep going to add it once the patch gets in and will address in a separate patch